<ui version="4.0" >
 <class>ConsoleForm</class>
 <widget class="QWidget" name="ConsoleForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Console</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTextEdit" name="textEdit" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>200</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement" >
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string comment="Console Window" />
     </property>
     <property name="statusTip" >
      <string comment="Console Window" />
     </property>
     <property name="whatsThis" >
      <string comment="Console Window" />
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="autoFormatting" >
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="tabChangesFocus" >
      <bool>false</bool>
     </property>
     <property name="documentTitle" >
      <string comment="Console Window" />
     </property>
     <property name="lineWrapMode" >
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionStatusDir" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/status.png</iconset>
   </property>
   <property name="text" >
    <string>StatusDir</string>
   </property>
  </action>
  <action name="actionConsoleHelp" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/utilities-terminal.png</iconset>
   </property>
   <property name="text" >
    <string>Console Help</string>
   </property>
  </action>
  <action name="actionRequestMessages" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/utilities-terminal.png</iconset>
   </property>
   <property name="text" >
    <string>Request Messages</string>
   </property>
  </action>
  <action name="actionConsoleReload" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/utilities-terminal.png</iconset>
   </property>
   <property name="text" >
    <string>Reload bacula-dir.conf</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
